crates/bevy_window/src/event.rs

```diff
@@ -3,6 +3,30 @@ use std::path::PathBuf;
 use super::{WindowDescriptor, WindowId};
 use bevy_math::{IVec2, Vec2};
 
+/// An event reporting that the ESC key has been pressed
```

Comment:
Why haven't you used <kbd>esc</kbd> (`<kbd>esc</kbd>`) to be consistent with `close_on_esc`?